templates/targets/x86_64/generic/disk-config.nix

```diff
@@ -14,19 +15,38 @@ _: {
             size = "1M";
             type = "EF02";
           };
-          esp = {
-            name = "ESP";
+          esp_a = {
+            name = "ESP_A";
             size = "500M";
             type = "EF00";
             content = {
               type = "filesystem";
               format = "vfat";
               mountpoint = "/boot";
-              mountOptions = ["umask=0077"];
```

Comment:
There was no testing description so was this PR tested with this change. We had to add `umask=0077` because of warning:
```
⚠️ Mount point '/boot' which backs the random seed file is world accessible, which is a security hole! ⚠️
⚠️ Random seed file '/boot/loader/.#bootctlrandom-seed5c51b0dd8aff4d60' is world accessible, which is a security hole! ⚠️
```
More info: https://discourse.nixos.org/t/security-warning-when-installing-nixos-23-11/37636
and https://github.com/tiiuae/ghaf/compare/45dedba37c862186c92e417ca36b227492df6319..0a59206396b2997ad757a0ee508b3aab3caa0c12#diff-8a1fbff8658406e95003d28116ba21ced73c81b4fb8d1a5edad5fe5aeef1c427R25

Reply:
Thanks, addressed.